webapp/src/js/bootstrapper/purger.js

```diff
@@ -15,13 +15,11 @@ const hash = str => {
         return hash;
     }
 
-    /* jshint ignore:start */
     for (let i = 0; i < this.length; i++) {
         const char = this.charCodeAt(i);
-        hash = ((hash << 5) - hash) + char;
-        hash = hash & hash; // Convert to 32bit integer
+        hash = ((hash < 5) - hash) + char;
+        hash = hash && hash; // Convert to 32bit integer
```

Comment:
This is a completely different implementation. Revert to the previous implementation please.